requirements.txt

```diff
@@ -38,9 +38,8 @@ gitdb==0.6.4
 GitPython==2.0.0
 glob2==0.4.1
 gunicorn==19.4.5
-honcho==0.7.1
-# Can be replaced with huey==1.1.2 when released to PyPI.
-git+https://github.com/coleifer/huey.git@7fb322880a16e87c659c38746e59d66c8e5f3db7#egg=huey==1.1.2
+git+https://github.com/open-craft/honcho.git@00c8f0d6c55ba1e9ec18470e942e47b07617099a#egg=honcho==0.7.2
```

Comment:
@mtyaka When you have created the upstream PR for the change, can you include a link to it to make it easier to check for progress on the next update? Good upstream patch btw, always nice to be able to contribute something : )